layers/meta-balena-beaglebone/recipes-core/images/balena-image-flasher.bbappend

```diff
@@ -2,6 +2,10 @@ include balena-image.inc
 
 BALENA_BOOT_PARTITION_FILES:append:beaglebone = " uEnv.txt_internal:"
 
+BALENA_BOOT_PARTITION_FILES:append:beagleplay = " \
```

Comment:
Hi @delhiryder this commit can be removed completely since @acostach worked out in https://github.com/balena-os/balena-beaglebone/pull/686/commits/1eae4da2350cf894680e37f697f33113ba34ebda that we can skip using extlinux.conf

Reply:
OK @floion